package.json

```diff
@@ -17,8 +17,8 @@
     "ci": "yarn test",
     "dev": "DEBUG=info,warn,error babel-node --extensions '.ts,.js' --inspect index.js",
     "dump-schema": "babel-node --extensions '.ts,.js' ./scripts/dump-schema.ts",
-    "dump:local": "USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA=false yarn dump-schema _schemaV2.graphql & wait",
-    "dump:staging": "node scripts/dump-staging-schema.js",
+    "dump:local": "USE_UNSTITCHED_ARTIST_SERIES_SCHEMA=true USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA=false yarn dump-schema _schemaV2.graphql & wait",
```

Comment:
Can undo this and the schema file update, but as the unstitching is done in one PR here _and_ there are no breaking changes to the schema (🤞 ) - this might be fine to do - get the new schema in front of clients.